kickstarts/partials/post/ui_compile.ks.erb

```diff
@@ -1,8 +1,6 @@
 npm install -g yarn
 
-<% if File.exist?(@build_base.join(KS_PART_DIR, "post", "npm_registry.ks.erb")) %>
-  <%= render_partial "post/npm_registry" %>
-<% end %>
+<%= render_partial_if_exist("/build/optional_kickstart_partials/npm_registry.ks.erb") %>
```

Comment:
I probably changed the directory name "optional_kickstart_partials" 3-4 times and kept the longest but most descriptive name here... I'm open for suggestions.